### ISSUE TITLE
Error Code: E-05-EXIUG 

### ISSUE THREAD

**Issue by github-fanuser** [Tech-FZ/EmuGUI#79]
I discovered this emulator. When I clicked New Virtual Machine, i get E-05-EXIUG error code. 
I have qemu-system-i386.exe and qemu-system-x86_64.exe set in the paths. Those are the only ones I have set. I only want to set up Windows Virtual Machines. 

I am running qemu on Windows 10 Pro 64bit

**Comment by github-fanuser** [comment]
Update - i figured out what was wrong. I had to enter the full path. My problem now is that the VM won't start. How do I boot from an iso to install OS? My hard drive image is named w98.vhd and the vhd format is raw. I don't know if this correct.

**Comment by Tech-FZ** [comment]
The file ending should be .img.

**Comment by github-fanuser** [comment]
Yeah. I figured that out. I now can get it to accept boot settings but nothing seems to happen.  i select the iso and put it in cd 1  and boot d. It is probably a setting that is incorrect, but there are no error message that I can see.

**Comment by github-fanuser** [comment]
Update - I got it working I had to reinstall QEMU. I just installed all components. It is installing Windows 98 now. I hope I can play some my old cdrom games. If I can, the next step is to install Windows 7. Is there anything special I should know? Does QEMU support 3D acceleration in Windows 7.  If I am able to get these OSes running, I will replace VMware. Thank You. This addon made it easy to setup Virtual Machines. I do not know Linux.

**Comment by github-fanuser** [comment]
Ok I have Win 98 running. There are a couple of problems. 1) No sound; I may have a a couple of ideas how to fix it.  2) Video is set at 640x480 with 16 Colors. No way to change it. Do you know how to fix it? I know there is a VGA setting in QEMU when you create the VM. I have already tried a couple of the options; no change. Is there a way to fix it?

**Comment by Tech-FZ** [comment]
To summon up the answers you're looking for:

> Ok I have Win 98 running.
Great job.

> No sound; I may have a a couple of ideas how to fix it.
The `sb16` sound card should work, if it doesn't, please tell me.

> Video is set at 640x480 with 16 Colors. No way to change it.
For Windows 98, the `cirrus` VGA should solve the issue. Again, if it doesn't, please tell me.

> Does QEMU support 3D acceleration in Windows 7.
Unless you switch to Linux, get a dedicated graphics card, a second mouse, a second keyboard and a second monitor, tell your PC via the BIOS it has to use the integrated one (without that integrated graphics, it won't work), do some complicated stuff EmuGUI doesn't even support (yet) and somehow manage to get the appropriate graphics drives for Windows 7, it doesn't.

**Comment by github-fanuser** [comment]
Thanks. I'm going to look around; maybe I can find something better.

**Comment by github-fanuser** [comment]
Another update - I finally got Win98 running with the proper display setting and sound. One last hurdle; I need to add a second hard drive, so that when I boot windows 98, I will have 2 hard drives. The hard drive image already exists.

Info:  98NewDisk.qcow2; 10GB; fat32; Vmware compatible (Vmware can see the contents of the image. Is there a way to do this in emugui or qemu? if not, is there a command or tool that will increase the size of the OS image. I don't wat to have to reinstall Win98 since I now have it working. Thanks! I really appreciate all your help!

**Comment by Tech-FZ** [comment]
EmuGUI currently doesn't support the usage of two hard drives. However, I can implement it into EmuGUI 2.1.

**Comment by github-fanuser** [comment]
Cool. It would be nice if you could add 3d acceleration as well.

Currently, I would like to play Law and Order: Justice is Served on QEMU.
The only emulator I find that can play this game is VMware player. There's
an option "3D acceleration in the vm options that has to be checked.

I want to move away from VMware...too bloated.

**Comment by Tech-FZ** [comment]
For 3D acceleration, there would be a few hurdles for both of us:
1. I have to implement support for assigning a dedicated graphics card (which is only possible on Linux btw and is really complicated)
2. You have to have an integrated AND a dedicated graphics card built into your computer (laptops don't count).
3. You need to be able to tell your computer to use your integrated graphics.
4. You need to have at least a second keyboard and a second mouse. A second monitor isn't really necessary, but I totally recommend it.
5. You need to switch to Linux because Windows is unable to assign random graphics cards to any VM.
6. You need to get the appropriate drivers for your graphics card and operating system (e. g. assigning an RTX 4080 to a Windows 7 VM is a bad idea).
